src/Runtime/LegacyServiceInstantiator.php

```diff
@@ -55,12 +57,58 @@ public function loadServiceFiles(array $serviceFiles)
                 $serviceFileData = Yaml::parse($serviceFileContents);
             }
 
-            if (isset($serviceFileData['services'])) {
+            if ($this->isValidServiceData($serviceFile, $serviceFileData)) {
                 $this->instantiateServices($serviceFileData['services']);
             }
         }
     }
 
+    /**
+     * Validate service data before using it.
+     *
+     * @param string $serviceFile Path to service file being checked
+     * @param array $serviceFileData Parsed data from drush.services.yml
+     */
+    protected function isValidServiceData(string $serviceFile, array $serviceFileData): bool
+    {
+        // If there are no services, then silently skip this service file.
+        if (!isset($serviceFileData['services'])) {
+            return false;
+        }
+
+        // We don't support auto-wiring
+        if (!empty($serviceFileData['services']['_defaults']['autowire'])) {
+            $this->logger->warning(dt('Autowire not supported; skipping @file', ['@file' => $serviceFile]));
```

Comment:
I think we should downgrade these to info(). They are typically going to be from commandfiles that you dont care about. Its not like user is trying to run a command in the file.